components/event-publisher-proxy/pkg/handler/handler.go

```diff
@@ -331,10 +330,9 @@ func (h *Handler) sendEventAndRecordMetrics(ctx context.Context, event *cev2even
 			originalEventType = event.Type()
 		}
 	}
-	h.collector.RecordEventType(originalEventType, event.Source(), result.HTTPStatus())
-	h.collector.RecordBackendLatency(duration, result.HTTPStatus(), host)
-	h.collector.RecordBackendRequests(result.HTTPStatus(), host)
```

Comment:
Dont we need this metric `h.collector.RecordBackendRequests(result.HTTPStatus(), host)`?

like `h.collector.RecordBackendRequests(http.StatusNoContent, host)`?

Reply:
BackendRequests is the total count of requests done to the backend -> 
eventing_epp_backend_duration_milliseconds_count

as the latency metric has destination_service and code as labels we can filter by those and get:
- all successful requests (filter by 2XX)
- all failed requests (filter by non2XX)
- all requests (dont filter)